src/domain/cohort.js

```diff
@@ -4,28 +4,47 @@ import dbClient from '../utils/dbClient.js'
  * Create a new Cohort in the database
  * @returns {Cohort}
  */
-export async function createCohort() {
+export async function createCohort(name) {
   const createdCohort = await dbClient.cohort.create({
-    data: {}
+    data: { name: name }
   })
 
-  return new Cohort(createdCohort.id)
+  return new Cohort(createdCohort.id, createdCohort.name)
 }
 
 export async function getAllCohorts() {
   const foundCohorts = await dbClient.cohort.findMany({})
   return foundCohorts
 }
 
+export async function getCohortById(id) {
+  const foundCohort = await dbClient.cohort.findUnique({
+    where: { id },
+    include: { deliveryLogs: true, users: true }
```

Comment:
need to include user profle and only select email and id from user as the password hash does not need to be sent

Reply:
added